app/src/main/java/com/rob729/quiethours/Activity/MainActivity.kt

```diff
@@ -24,11 +25,13 @@ class MainActivity : AppCompatActivity() {
         )
         supportActionBar?.elevation = 0F
         val appSharedPrefs = PreferenceManager.getDefaultSharedPreferences(applicationContext)
-        if (appSharedPrefs.getBoolean("nightMode", false)) {
+        if (appSharedPrefs.getBoolean("nightMode", StoreSession.readBoolean(AppConstants.IS_DARK))) {
```

Comment:
what is the use of AppConstants.IS_DARK shared pref when "nightMode" is already there?
Use only "nightMode" shared pref and implement the same.
Also, improve the code used for "nightMode" shared pref like other shared prefs used in the app.

Reply:
@rob729 I am able to use nightMode in place of IS_DARK. Just one question, while improving the code like other shared prefs, appSharedPrefs is of type getDefaultSharedPreferences, whereas StoreSession is taking getSharedPreferences. So where shall I put it?

Reply:
@rob729 I have made the changes that were asked. Please have a look